types/dialog-node/index.d.ts

```diff
@@ -0,0 +1,129 @@
+// Type definitions for dialog-node 0.2
+// Project: https://github.com/bat-tomr/dialog-node
+// Definitions by: Ian MacLeod <https://github.com/DefinitelyTyped>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Information dialog with text and title and an OK button.
+ *
+ * @param msg The message to be displayed in the body of the dialog.
+ * @param title Displayed in the title of the dialog.
+ * @param timeout number of seconds before timing out the dialog. Set to 0 for no timeout.
+ * @param callback called when the dialog closes.
+ */
+export function info(
+	msg: string,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/dialog-node/index.d.ts;
  Line: 15;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.